src/dataframe.js

```diff
@@ -851,19 +851,33 @@ class DataFrame {
     }
 
     /**
-     * Sort DataFrame rows based on a column values. The row should contains only one variable type.
-     * @param {String} columnName The column giving order.
+     * Sort DataFrame rows based on column values. The row should contains only one variable type.
+     * @param {String | Array<string>} columnNames The columns giving order.
      * @param {Boolean} [reverse=false] Reverse mode. Reverse the order if true.
      * @returns {DataFrame} An ordered DataFrame.
      * @example
      * df.sortBy('id')
-     */
-    sortBy(columnName, reverse = false) {
+     * df.sortBy(['id1', 'id2'])
+     * df.sortBy(['id1'], true)
+    */
+    sortBy(columnNames, reverse = false) {
+        let _columnNames = columnNames;
+        if (!Array.isArray(_columnNames)) { _columnNames = [_columnNames]; }
+
+        // ensure unique columns
+        const tmp = new Set(_columnNames);
+        _columnNames = Array.from(tmp);
+
         const sortedRows = this[__rows__].sort((p, n) => {
-            const [pValue, nValue] = [p.get(columnName), n.get(columnName)];
-            if (typeof pValue !== typeof nValue) { throw new MixedTypeError(); }
-            return compare(pValue, nValue, reverse);
+            return _columnNames.map((col) => {
+                const [pValue, nValue] = [p.get(col), n.get(col)];
+                if (typeof pValue !== typeof nValue) { throw new MixedTypeError(); }
+                return compare(pValue, nValue, reverse);
+            }).reduce((acc, curr) => {
```

Comment:
Ok it seems good.

Reply:
It could be important to indicate in the documentation that column names order (and not the order in the schema) is kept in account to sort values (left to right strategy).